### PR TITLE
Allow the application to start up even if the database service env var i...

### PR DESCRIPTION
**Pull request by brenton**
...s missing

This is an approach at what @thoraxe wanted.  I tested a deployment with and without the database and it worked as expected.  I'm not exactly sure how the database service will be added after the fact but this should be enough to unblock the beta2 testing.  Once we've settled on the approach we can merge this (or something similar) in.

### CODE REVIEW

**Comment by bparees** [comment]
Thanks @brenton, you rock! :)  this looks good with one caveat...  the DB service env could also be named DATABASE_TEST_SERVICE_HOST (technically the app supports two potential DB services).  Can you change the logic to check for either of them?

@mfojtik @jhadvig you can ignore my previous request.

**Comment by jhadvig** [comment]
@brenton :+1:

**Comment by brenton** [comment]
PR updated.  Let me know if this looks good.  Thanks for the review!

File: views/main.erb [hunk 1]
```diff
@@ -18,6 +18,11 @@
     <h1> Welcome to an OpenShift v3 Demo App! </h1>
   </div>
 
+  <% if ENV["DATABASE_SERVICE_HOST"].nil? %>
```
Comment (bparees) [inline views/main.erb]: you need to check both env vars here too.

Comment (brenton) [inline views/main.erb]: Yikes... sorry about that.  Updated!

**Comment by bparees** [comment]
Thanks again!

**Pull request merged**